app/assets/javascripts/admin/addon/components/dashboard-new-features.gjs

```diff
@@ -0,0 +1,30 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { ajax } from "discourse/lib/ajax";
+import DashboardNewFeatureItem from "admin/components/dashboard-new-feature-item";
+
+export default class DashboardNewFeatures extends Component {
+  @tracked newFeatures = null;
+
+  constructor() {
+    super(...arguments);
+
+    ajax("/admin/dashboard/new-features.json").then((json) => {
+      if (this.isDestroying || this.isDestroyed) {
+        return;
+      }
+
+      this.newFeatures = json.new_features;
+    });
+  }
+
+  <template>
+    {{#if this.newFeatures}}
```

Comment:
We need to show something when there are no features; Ella suggested showing a link to the announcements on Meta. I think something something simple like `You can see previous new feature announcements on [Discourse Meta](https://meta.discourse.org/tags/c/announcements/67/release-notes)` , since it shouldn't be seen that often. This would also handle any AJAX errors.